<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindQQDialog</class>
 <widget class="QDialog" name="FindQQDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>捕获QQ号码</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QTreeWidget" name="findTreeWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>821</width>
     <height>371</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>首次捕获时间</string>
    </property>
    <property name="textAlignment">
     <set>AlignHCenter|AlignVCenter|AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>最后捕获时间</string>
    </property>
    <property name="textAlignment">
     <set>AlignHCenter|AlignVCenter|AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>来源 IP 地址</string>
    </property>
    <property name="textAlignment">
     <set>AlignHCenter|AlignVCenter|AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>目标 IP 地址</string>
    </property>
    <property name="textAlignment">
     <set>AlignHCenter|AlignVCenter|AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>QQ号码</string>
    </property>
    <property name="textAlignment">
     <set>AlignHCenter|AlignVCenter|AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>捕获次数</string>
    </property>
    <property name="textAlignment">
     <set>AlignHCenter|AlignVCenter|AlignCenter</set>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="beginPushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>420</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>开始捕获</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="endPushButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>420</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>结束捕获</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitPushButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>420</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="infoPushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>420</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>工具简介</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>841</width>
     <height>401</height>
    </rect>
   </property>
   <property name="title">
    <string>捕获结果：</string>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>findTreeWidget</zorder>
  <zorder>beginPushButton</zorder>
  <zorder>endPushButton</zorder>
  <zorder>exitPushButton</zorder>
  <zorder>infoPushButton</zorder>
 </widget>
 <tabstops>
  <tabstop>infoPushButton</tabstop>
  <tabstop>beginPushButton</tabstop>
  <tabstop>endPushButton</tabstop>
  <tabstop>exitPushButton</tabstop>
  <tabstop>findTreeWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>exitPushButton</sender>
   <signal>clicked()</signal>
   <receiver>FindQQDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>389</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>387</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
